content/reference-data-api.md

```diff
@@ -3006,6 +3212,190 @@ Response:
 
 
 
+## Get Account Transaction Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
```

Comment:
Not quite. The default is an array of daily summaries covering the entire active history of the account. The query parameters can filter it down to fewer total days, results, or reverse order, but it pretty much always returns "daily summaries of transaction activity for an account."